lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -94,27 +94,30 @@ class UsesResponsiveImages extends ByteEfficiencyAudit {
     const warnings = [];
     /** @type {Map<string, LH.Audit.ByteEfficiencyItem>} */
     const resultsMap = new Map();
-    images.forEach(image => {
-      // TODO: give SVG a free pass until a detail per pixel metric is available
-      if (!image.resourceSize || image.mimeType === 'image/svg+xml') {
-        return;
+    for (const image of images) {
+      // Ignore images without resource size information.
+      // Give SVG a free pass because creating a "responsive" SVG is of questionable value.
+      // Ignore CSS images because it's difficult to determine what is a spritesheet,
+      // and I mean just c'mon https://css-tricks.com/responsive-images-css/.
```

Comment:
I don't know what this comment means :)

Reply:
Oh it's calling out the hoops you have to jump through just to try and get super basic, not very helpful support for srcset-like behavior in CSS, happy to change to whatever sentence most embodies the idea that there are probably way more important bundley things we should be encouraging our users to look at before responsive CSS images :)

Reply:
ha, well I think some variation on that comment sounds good to me :)

Reply:
```suggestion
      // and the reward-to-effort ratio for responsive CSS images is quite low https://css-tricks.com/responsive-images-css/.
```